11-eos-storage.md

```diff
@@ -113,6 +116,9 @@ Once your job has completed and the files have been copied to
 EOS by `ganga` you can access them from your terminal by mounting
 your EOS area.
 
+Important: mounting eos is discouraged by the LHCb data management team,
+and no support will be provided by them for problems/questions related to it.
+
```

Comment:
Actually it was discussed in some presentations, maybe LHCbWeek somewhere. There are plenty of technical reasons, but well.... :-)